src/main/java/dk/kb/storage/api/v1/impl/DsStorageApiServiceImpl.java

```diff
@@ -123,8 +132,8 @@ public StreamingOutput getRecordsModifiedAfter(String origin, Long mTime, Long m
         }
     }
 
-    @Override
-    public StreamingOutput getRecordsByRecordTypeModifiedAfterLocalTree(String origin, RecordTypeDto recordType, Long mTime, Long maxRecords) {
+
+    private StreamingOutput getRecordsByRecordTypeModifiedAfterLocalTree(String origin, RecordTypeDto recordType, Long mTime, Long maxRecords) {
```

Comment:
As this is a private method now, maybe it should be moved to the facade?

Reply:
Not possible without a lot of refactoring since it modifies  httpServletResponse. The facade should now be involved with web-handling. But else it would be a good design decision.